### ISSUE TITLE
Snakecase field names containing a number are not fully converted to camelcase

### ISSUE THREAD

**Issue by jesselathamdev** [Kequc/knex-stringcase#24]
Hi there,

We've recently pulled knex-stringcase into a project of ours and are enjoying it so far, with a couple oddities discovered.

The one I'm raising here is with respect to field names which include numbers in them.  It appears as though if an underscore appears before a number then the underscore is not removed - although the rest of the string is snake to camel converted.

**Expected results:**
`address_line_1` is changed to `addressLine1`

**Actual results:**
`address_line_1` is changed to `addressLine_1`

Internally our app and API has `addressLine1` defined for all our schemas.  Our database as you can expect uses snakecase everywhere.

**Versions:**
* knex-stringcase: 1.4.6
* knex: 1.0.3
* postgresql v13
* node-postgres: 8.7.3
* node: 16.13.1
* npm: 8.1.2

**Comment by Kequc** [comment]
Oh wow that's really interesting. The library uses `"stringcase": "^4.3.1"` internally as it's only dependency, it is used for conversion between cases. I wonder if there is a newer version which fixes this oddity, or if there is a better library which would address this concern.

**Comment by jesselathamdev** [comment]
I had wondered if by using regex and look aheads it would fail to consider the last underscore as the number is the last character for the field name, or else the regex just fails to consider numbers throughout or in last place in general?

**Comment by Kequc** [comment]
I'm normally opposed to 3rd party libraries and dependencies, however in this case I opted to use one for this reason. So that I'm not accountable for inaccuracies in conversion between cases.

I am willing to switch to a better library. I just don't know very much about the differences, `case` might be a viable one. As it is the first result, and was updated more recently than `stringcase`.

More testing would be required.
